ui/src/app/pages/login/account-creation/account-creation.tsx

```diff
@@ -396,173 +452,197 @@ export class AccountCreation extends React.Component<AccountCreationProps, Accou
+                                    labelText='Country' containerStyle={styles.multiInputSpacing}
+                                    onChange={value => this.updateAddress('country', value)}/>
+              </FlexRow>
+            </FlexColumn>
+          </Section>
+          <Section header='Institutional Affiliation'>
+            <label style={{color: colors.primary, fontSize: 16}}>
+              Are you affiliated with an Academic Research Institution?
+            </label>
+            <div style={{paddingTop: '0.5rem'}}>
+              <RadioButton data-test-id='show-institution-yes'
+                           onChange={() => {this.clearInstitutionAffiliation();
+                             this.setState({showInstitution: true}); }}
+                           checked={this.state.showInstitution} style={{marginRight: '0.5rem'}}/>
+              <label style={{paddingRight: '3rem', color: colors.primary}}>
+                Yes
+              </label>
+              <RadioButton data-test-id='show-institution-no'
+                           onChange={() => {this.clearInstitutionAffiliation();
+                             this.setState({showInstitution: false}); }}
+                           checked={!this.state.showInstitution} style={{marginRight: '0.5rem'}}/>
+              <label style={{color: colors.primary}}>No</label>
             </div>
+          </Section>
+          {this.state.showInstitution &&
+          <FlexColumn style={{justifyContent: 'space-between'}}>
+            <TextInput data-test-id='institutionname' style={{width: '16rem', marginBottom: '0.5rem',
+              marginTop: '0.5rem'}}
+              value={institutionalAffiliations && institutionalAffiliations.length > 0 ?
+                  institutionalAffiliations[0].institution : ''}
+              placeholder='Institution Name'
+              onChange={value => this.updateInstitutionAffiliation('institution', value)}
+                       ></TextInput>
+            <Dropdown data-test-id='institutionRole' value={institutionalAffiliations &&
+            institutionalAffiliations.length > 0 ?
+                institutionalAffiliations[0].role : ''}
+                      onChange={e => this.updateInstitutionAffiliation('role', e.value)}
+                      placeholder='Which of the following describes your role'
+                      style={{width: '16rem'}} options={AccountCreationOptions.roles}/>
+          </FlexColumn>}
+          {!this.state.showInstitution &&
+          <FlexColumn style={{justifyContent: 'space-between'}}>
+            <Dropdown data-test-id='affiliation'
+                      style={{width: '18rem', marginBottom: '0.5rem', marginTop: '0.5rem'}}
+                      value={institutionalAffiliations && institutionalAffiliations.length > 0 ?
+                          institutionalAffiliations[0].nonAcademicAffiliation : ''}
```

Comment:
Hmm i think this (institutionalAffiliations && institutionalAffiliations.length > 0?..:..) be taken out in a function with a parameter as the property to return the 0th  parameter value if true else ''

Reply:
I didn't actually touch this code, I think it is an indentation change? I can change it if you'd like?